src/functions/exportFunctions.js

```diff
@@ -121,7 +125,47 @@ export const fetchAndTransformQuizzes = async (
   }
 };
 
-export async function fetchData(
+export const fetchAndAddUserQuizzes = async (
+  backendApiCall,
+  quizzes,
+  onSuccess,
+  onError,
+  setUserQuizzesUpdated,
+  auth
+) => {
+  if (!auth.loggedIn) return;
+
+  try {
+    const userQuizDataArray = await backendApiCall(
+      'GET',
+      `/api/v1/progress/user?userId=${auth.userId}`
+    );
+
+    const updatedQuizzes = quizzes.map((quiz) => {
+      const userQuizData = userQuizDataArray.data.find(
+        (uqd) => uqd.test === quiz.id
+      );
+      if (userQuizData) {
+        return {
+          ...quiz,
+          quizProgress: {
+            attemptsCount: userQuizData.attemptNumber.toString(),
+            bestScore: userQuizData.maxScore,
```

Comment:
I didn't really understand what the green progress bar indicated the first time I used this page. Was it the number of questions answered? Was it my latest score? My best score?

I think it'd be good to add a text label somewhere on the quiz card describing the user progress. Perhaps something like this? 
![image](https://github.com/Code-the-Dream-School/ee-prac-team3-front/assets/6400323/953aa1ca-2730-49d9-bf62-e69a3fde2efd)

Keep the progress bar! It's a great feature and is a great visual aid.

Reply:
I love this idea and I have implemented a version of it.  Since Kat is sort of our design guru, I will also let her do the final touches on the design, but there is something there for her to work with.